docs/tutorials/get-started/try-viam-sdk.md

```diff
@@ -38,39 +38,108 @@ If you are running out of time during your rental, you can [extend your rover re
 python3 square.py
 ```
 
 {{% /tab %}}
 {{% tab name="Go" %}}
 
-If using Go, you need to initialize your project, and install the necessary libraries before running the program.
+The easiest way to get started writing an application with Viam is to navigate to the [robot page on the Viam app](https://app.viam.com/robots), select the **code sample** tab, and copy the boilerplate code from the section labeled **Go**.
+
+This code snippet imports all the necessary libraries and sets up a connection with the Viam app in the cloud.
+
+Next, create a file named <file>square.go</file> and paste the boilerplate code from the **code sample** tab of the Viam app into your file.
+Then, save your file.
+
+Initialize your project, and install the necessary libraries, and then run the program to verify that the Viam SDK is properly installed and that the `viam-server` instance on your robot is live:
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 go mod init square
 go mod tidy
 go run square.go
 ```
 
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+The easiest way to get started writing an application with Viam is to navigate to the [robot page on the Viam app](https://app.viam.com/robots), select the **code sample** tab, and copy the boilerplate code from the section labeled **TypeScript**.
+
+This code snippet imports all the necessary libraries and sets up a connection with the Viam app in the cloud.
+
+Next, create a file named <file>main.ts</file> and paste the boilerplate code from the **code sample** tab of the Viam app into your file.
+Then, save your file.
+
+Create another file named <file>package.json</file> with the following contents:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "name": "test-rover",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "watch": "esbuild ./main.ts --bundle --outfile=static/main.js --servedir=static",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Viam Docs Team",
+  "license": "ISC",
+  "devDependencies": {
+    "esbuild": "0.16.12"
+  },
+  "dependencies": {
+    "@viamrobotics/sdk": "^0.0.28"
+  }
+}
+```
+
+Create a folder <file>static</file> and inside it another file named <file>index.html</file>.
+Add the following markup:
+
+```html {class="line-numbers linkable-line-numbers"}
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Drive a Viam Rover</title>
+    <link rel="icon" href="favicon.ico" />
+  </head>
+  <body>
+    <div id="main">
+      <button id="main-button" disabled=true>
+        Click me
+      </button>
+    </div>
+    <script type="module" src="main.js">
+    </script>
+  </body>
+</html>
+```
+
+Run the following commands to install the necessary libraries, and then run the program to verify that the Viam SDK is properly installed and that the `viam-server` instance on your robot is live:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+npm install
+npm run watch
```

Comment:
if you agree with [this change](https://github.com/viamrobotics/docs/pull/1130/files#r1180720627) then:

```suggestion
npm start
```